### PR TITLE
Allow multiple datasets so you can create a dictionary from train/test

### PR DESCRIPTION
**Pull request by ymt123**

### CODE REVIEW

File: attalos/dataset/dataset.py [hunk 1]
```diff
@@ -67,27 +87,21 @@ def get_next_batch(self, batch_size):
...
-                for feat in self.text_feats[item_id]:
-                    feat = str(feat)
-                    text_feat.append(self.tag_transformer[feat])
-            else:
-                text_feat = self.text_feats[item_id]
+            img_feats[i, :] = img_feat
             text_feats.append(text_feat)
         return img_feats, text_feats
 
+    def get_num_imgs(self):
```
Comment (agude) [inline attalos/dataset/dataset.py]: There is an argument to be made (for example: https://stackoverflow.com/questions/17330160/how-does-the-property-decorator-work/17330273#17330273) that `@property` is preferable to getters and setters, but I have no strong opinion on it.

**Comment by agude** [comment]
Looks good, I'm happy to merge if you're happy with the `getters()` and `setters()`.

**Comment by ymt123** [comment]
@agude Updated per suggestion

**Comment by agude** [comment]
Can we get an `text_feat_size` variable in the class? There is a `img_feat_size` which is great for setting the size of layers in the network.

**Comment by ymt123** [comment]
@agude I think what you'd really want here is the onehot/word2vec vector size which isn't really available to the dataset class. I've renamed the "num_keys" in onehot to "vocab_size".

**Comment by agude** [comment]
@ymt123 Quite right! Looks good and runs fine with my tensorflow code.

**Pull request merged**